### DISCUSSION TITLE
Can kiwi browser render code-server?

### DISCUSSION THREAD

**Discussion by coolcodehere** [coder/code-server#2257] [Q&A]
I'm running code-server on an emulated version of ubuntu from android on a Samsung Tab S7+ and Kiwi browser doesn't render anything when the server is opened. Screenshot of Chrome(left) and Kiwi(right) both with vscode open from code-server. Any fix for this? 
https://i.imgur.com/zaHG9Dc.png

Is this a bug or am I doing something wrong?

**Comment by code-asher** [comment]
Would you be able to open Kiwi's browser console and post the logs from
there? I'm not completely sure how you do this on Android though, you
might have to connect a remote debugger from a PC or install a browser
extension that lets you view the console.

**Comment by sr229** [comment]
Kiwi seems to be using an ancient engine. VSCode requires most if not all HTML5 features. Consider getting alternatives instead.
